### PR TITLE
Azure security center - update regulatory compliance

### PR DESCRIPTION
**Pull request by himarkov**
Swagger link: https://github.com/Azure/azure-rest-api-specs/blob/master/specification/security/resource-manager/Microsoft.Security/preview/2019-01-01-preview/regulatoryCompliance.json

### CODE REVIEW

**Comment by dsgouda** [comment]
Please fix the merge conflicts

File: src/SDKs/SecurityCenter/Management.SecurityCenter/Generated/RegulatoryComplianceAssessmentsOperations.cs [hunk 1]
```diff
@@ -122,7 +122,7 @@ public async Task<AzureOperationResponse<IPage<RegulatoryComplianceAssessment>>>
             }
             // Construct URL
             var _baseUrl = Client.BaseUri.AbsoluteUri;
-            var _url = new System.Uri(new System.Uri(_baseUrl + (_baseUrl.EndsWith("/") ? "" : "/")), "subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Security/regulatoryComplianceStandards/{regulatoryComplianceStandardName}/regulatoryComplianceControls/{regulatoryComplianceControlName}/regulatoryComplianceAssessments").ToString();
```
Comment (dsgouda) [inline src/SDKs/SecurityCenter/Management.SecurityCenter/Generated/RegulatoryComplianceAssessmentsOperations.cs]: Were these files manually modified? It doesn't look like the changes were generated by Autorest

**Comment by dsgouda** [comment]
Gentle ping

**Comment by dsgouda** [comment]
Ping, this PR will be closed by EOW if there's no activity

**Comment by erich-wang** [comment]
@himarkov, this PR is closed since there's no response from your side, please open a new PR if necessary.

**Pull request closed without merging**